services/reference/zksync/json-rpc-methods/_eth_accounts-request.mdx

```diff
@@ -1,15 +1,23 @@
-import Tabs from "@theme/Tabs"
-import TabItem from "@theme/TabItem"
+import Tabs from '@theme/Tabs';
```

Comment:
@bgravenorst we have been using double quotes for our docs. Is that okay?

Reply:
Yep, I'll keep an eye out for that next time.